app/routes.test.js

```diff
@@ -0,0 +1,389 @@
+import request from 'supertest';
+import { App } from '../app';
+import routes from './routes';
+import * as dashboardControllers from './pages/dashboard/controller';
+import * as sectionControllers from './pages/section/controller';
+import * as subsectionControllers from './pages/dashboard/subDashboards/controller';
+import * as previewControllers from './pages/preview/controller';
+
+jest.mock('./logger');
+
+const mockDashboardContext = {
+  title: 'Dashboard title',
+  previewUrl: '/',
+  submitForModerationUrl: '/',
+  returnToDashboardUrl: '/',
+  sectionGroups: [
+    {
+      id: 'sectiongroup-id',
+      title: 'Sectiongroup Title',
+    },
+  ],
+};
+
+const mockSectionContext = {
+  title: 'Section Title',
+  submitActionUrl: '/',
+  mainAdvice: 'Section Main Advice',
+  additionalAdvice: [
+    'Section additional advice',
+  ],
+  returnToDashboardUrl: '/',
+  submitText: 'submitText',
+  questions: [
+    {
+      id: 'a-question',
+      type: 'bulletpoint-list',
+    },
+  ],
+};
+
+const mockSectionPostData = {
```

Comment:
last one I promise. Since we are mocking this. Could we mock a nice question type than bullet-list please. So we don't get all them `''` in there

Reply:
sure